workers/src/main/scala/workers/itc/ITCRequests.scala

```diff
@@ -53,9 +53,10 @@ object ITCRequests:
     cache:               Cache[F],
     callback:            Map[ItcRequestParams, EitherNec[ItcTargetProblem, ItcResult]] => F[Unit]
   )(using Monoid[F[Unit]], ItcClient[F]): F[Unit] = {
-    def itcResults(r: ClientCalculationResult): EitherNec[ItcTargetProblem, ItcResult] =
+    def itcResults(r: ClientAllResults): EitherNec[ItcTargetProblem, ItcResult] =
```

Comment:
This will return a `NonEmptyList[EitherNec[ItcTargetProblem, ItcResult]]`?

Reply:
Yes, but that will be the behavior for multiple modes, right? Just loop through the `all`?

Reply:
Yes I expect that you'd loop over all the `modes`